cmake/windows/compilerconfig.cmake

```diff
@@ -53,6 +57,9 @@ endif()
 add_compile_options(
   /W3
   /utf-8
+  /permissive-
```

Comment:
Clang seems to supports this. Ignore the mention of a revert. They didn't actually revert it, and simply added a release note item.
https://reviews.llvm.org/D103773